04.ls/wc.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+OPTION_FLAG_TO_STAT_KEY = {
+  l: :lines,
+  w: :words,
+  c: :bytes
+}.freeze
+
+def main
+  option_flag = parse_options
+  if !ARGV.empty?
+    process_files(option_flag)
+  elsif !$stdin.tty?
+    print_stats_from_stdin(option_flag)
+  else
+    exit
+  end
+end
+
+def parse_options
+  opt = OptionParser.new
+  params = { l: false, w: false, c: false }
```

Comment:
```suggestion
  params = { lines: false, words: false, bytes: false }
```

この時点でキーを揃えてしまえば、OPTION_FLAG_TO_STAT_KEYは不要なんじゃないかなと思いました

Reply:
https://github.com/outa1127/ruby-practices/pull/9/commits/ac12f060a8a7ed92e4ae6918dbf0721921f2dfe0 にて修正いたしました。
`OPTION_FLAG_TO_STAT_KEY`のハッシュを削除したことに伴い、ハッシュを使用して繰り返し処理を行なっていた箇所を複数修正しております。